.gitignore

```diff
@@ -8,3 +8,5 @@ Cargo.lock
 /cli/trusted_setup
 .cargo/config
 .DS_Store
+.vscode/
+~/
```

Comment:
why `~/`?

Reply:
In the process of building the code, git always includes some library file changes (~/) and vscode configuration file changes, so I can’t clearly see the current project changes through git, so I put the library files in vscode The configuration file is added to gitignore.
~/.cargo is my cargo library.